manager/allocator/networkallocator/networkallocator.go

```diff
@@ -87,8 +87,8 @@ type NetworkAllocator interface {
 	// DeallocateLBAttachment Deallocates a load balancer endpoint for the node
 	DeallocateLBAttachment(node *api.Node, networkAttachment *api.NetworkAttachment) error
```

Comment:
Is leaving this and the `AllocateLB` just above deliberate?

Reply:
Nope. I'll hunt these down.